TestHelpers/struct_field_to_index_map_generator.py

```diff
@@ -0,0 +1,312 @@
+#! python3 struct_field_to_index_map_generator.py <ast txt file> <output for map file>
```

Comment:
In a separate PR, you should add an autoformatter for the python code in this repo. I suggest [black](https://github.com/psf/black), but any autoformatter will do. (For a discussion of the options, [this post](https://www.locallyoptimal.com/blog/2019/08/23/why-you-should-use-black-for-your-python-style-linting/) is very good. I have my students in CS 490 read this before our discussion of autoformatting, because the rationales it discusses are quite reasonable and reflect good engineering practice.)

Reply:
I have installed a formatter in the `test-suite-qol-additions` branch and have isolated the formatter into the `python-formatter` branch; it uses autopep8. I'll make the PR for it in a bit